src/content/release-notes.mdx

```diff
@@ -5,6 +5,18 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.27.2
+
+10 January 2025
+
+This version is compatible with Kubernetes versions 1.27 to 1.30 \
+Okteto Chart release 1.27 is designed to work with [Okteto CLI 3.2.x](https://github.com/okteto/okteto/releases/tag/3.2.1)
+
+### Bug Fixes
+
+- Docker compose no longer times out when waiting for dependant service to be ready
+- Removes the init container security context to use the same as the original image 
```

Comment:
When they are items related to the CLI, we usually mention it explicitly like in [this case](https://github.com/okteto/docs/blob/main/src/content/release-notes.mdx?plain=1#L48) for example